halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/converter/LogLevelConverter.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2017 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.netflix.spinnaker.halyard.cli.command.v1.converter;
+
+import ch.qos.logback.classic.Level;
+import com.beust.jcommander.IStringConverter;
+
+public class LogLevelConverter implements IStringConverter<Level> {
+  @Override
+  public Level convert(String value) {
+    Level def = Level.OFF;
+    Level res = Level.toLevel(value, Level.OFF);
+
+    if (!def.levelStr.equalsIgnoreCase(value)) {
```

Comment:
I would if it was an enum, those `OFF`, `INFO` etc.. are fields in a class: https://logback.qos.ch/apidocs/ch/qos/logback/classic/Level.html